content/engineering/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Guidelines and Best Practices
+permalink: /engineering/
+layout: layouts/page
+sidenav: true
+tags: engineering
+eleventyNavigation: 
+  parent: engineering
+  key: Guidelines and Best Practices
+  order: 1
+  title: Guidelines and Best Practices
+---
+This guide is where the TTS Engineering Practices Guild collects its best practices and resources for software development at TTS, as well as on our partner engagements. Our focus is cloud-native digital services and our recommendations in this guide reflect the needs of that domain.
+
+{% comment %}
+I am not sure if this will work, something to test once I get all other files in place
```

Comment:
I am going to leave it because if you see in the production, it list all the sections (https://engineering.18f.gov/). Do we have a way of listing it? I do need to make a ticket out of this. @igorkorenfeld

Reply:
I am going to put this as part of #338 (I have updated the ticket)

Reply:
We can either do this manually, or we can maybe leverage the `eleventyNavigation`. Resolving as part of #338 makes sense to me, and we can explore it as part of that follow-on work.